doc/manuscript.Rmd

```diff
@@ -72,28 +72,206 @@ pretty <- function(x) format(x, big.mark = ",")
 
 # Results
 
-The aggregated data-set describes `r nrow(data_clean) |> pretty()` observations of daily physical activity and sleep from `r length(unique(data_clean$participant_id)) |> pretty()` unique participants. Table \@ref(demo) shows demographic information. A table of study characteristics can be found in supplementary materials.
+The aggregated data-set describes `r nrow(data_clean) |> pretty()` observations of daily physical activity and sleep from `r length(unique(data_clean$participant_id)) |> pretty()` unique participants. Table \@ref(demo) [Mike's table to be inserted] shows demographic information. A table of study characteristics can be found in supplementary materials.
```

Comment:
Something to consider (but I don't feel strongly about) is that it's sometimes faster to have a intermediate target that just makes a big list of stuff that you need to knit the manuscript. E.g., it calculated `nrow(data_clean)`. Then you only load the intermediate target in the manuscript.

The reason I do that is that it means that you'd don't have to load the full dataset every time you want to knit, and you tend to make way more changes to the manuscript text than the things that are in the list.

But it doesn't really matter - we could look at adding it later.